pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation.go

```diff
@@ -915,3 +926,12 @@ func dotnetProfilingLdPreloadEnvValFunc(predefinedVal string) string {
 func rubyEnvValFunc(predefinedVal string) string {
 	return predefinedVal + rubyOptValue
 }
+
+func isTruthy(value string) bool {
```

Comment:
In `cmd/serverless-init/log/log.go`:

```
func isEnabled(envValue string) bool {
	return strings.ToLower(envValue) == "true"
}
```

We should probably use a similar definition of valid values for true here, in addition to 1.

Looking at the .NET tracer library, 1 seems to be the expected value but maybe @andrewlock can confirm.

Reply:
> Looking at the .NET tracer library, 1 seems to be the expected value

The short answer is that we are not _always_ consistent, but native code is where we support the fewest versions, and we support these: https://github.com/DataDog/dd-trace-dotnet/blob/08c9c4cda7ea7e4b53fe7e1e8d194fe2a0164769/shared/src/native-src/util.cpp#L82

In managed code we're more lax in that we use case-insensitive comparisons: https://github.com/DataDog/dd-trace-dotnet/blob/88242c571be849798b578e1ed7f77f5cb9543629/tracer/src/Datadog.Trace/ExtensionMethods/StringExtensions.cs#L68-L77